pol-core/pol/realms/realm.h

```diff
@@ -214,6 +226,28 @@ class Realm
   }
   void readmultis( Plib::StaticList& vec, const Core::Pos2d& pos ) const;
 
+  void readdynamics( Plib::MapShapeList& vec, const Core::Pos2d& pos,
+                     Core::ItemsVector& walkon_items, bool doors_block, unsigned int flags ) const;
+
+  void readdynamics( Plib::MapShapeList& vec, unsigned short x, unsigned short y,
+                     Core::ItemsVector& walkon_items, bool doors_block, unsigned int flags ) const
+  {
+    return readdynamics( vec, Core::Pos2d( x, y ), walkon_items, doors_block, flags );
+  }
+
+  void readdynamics( Plib::MapShapeList& vec, unsigned short x, unsigned short y,
+                     Core::ItemsVector& walkon_items, bool doors_block ) const
+  {
+    return readdynamics( vec, Core::Pos2d( x, y ), walkon_items, doors_block,
+                         Plib::FLAG::WALKBLOCK );
+  }
+
+  void readdynamics( Plib::MapShapeList& vec, const Core::Pos2d& pos,
+                     Core::ItemsVector& walkon_items, bool doors_block ) const
+  {
+    return readdynamics( vec, pos, walkon_items, doors_block, Plib::FLAG::WALKBLOCK );
```

Comment:
I would give the function a different name, it's unclear only due to slightly different arguments that WALKBLOCK is here used